src/send/SendAmount/useTransactionCallbacks.ts

```diff
@@ -103,7 +103,7 @@ function useTransactionCallbacks({
 
   const feeType = FeeType.SEND
   const estimateFeeDollars =
-    useSelector(getFeeEstimateDollars(feeType, transferTokenAddress)) ?? new BigNumber(0)
+    useSelector(getFeeEstimateDollars(feeType, tokenInfo!.address!)) ?? new BigNumber(0)
```

Comment:
another non-null assertion operator. please remove it and deal with the case where tokenInfo is null instead.

also, it's weird to me that this useTransactionCallbacks hook has been refactored to accept tokenId as a parameter, but then uses token addresses under the hood, breaks with a runtime error if an address does not exist, and seems to assume every token's address is a Celo address (here, when getting estimated fees). 

Unfortunately, it looks like refactoring `getFeeEstimateDollars` to use token ID's is going to take some effort. But leaving the hook in this half-refactored state seems very risky

Reply:
your Linear issue says actually issuing the send and doing fee estimation for ethereum is out of scope. maybe you could avoid changing this hook at all.. you'd need to throw an error from SendAmount if "review" is pressed and the network id is not celo or address was undefined, and leave a todo to pick it up in a later issue in the epic, but that's probably better than silently failing like this imo

Reply:
`Send Review` refers  to `src/send/SendConfirmation.tsx` which doesn't correctly render at present; `src/send/SendAmount/index.tsx` is the previous screen in the flow. It seems like updating this screen work will take keeping the updates to useTransactionCallbacks.ts and also updating `TransactionDataInput` type. Would it be better to revert the changes to the `useTransactionCallbacks.ts` and punt on the display for `src/send/SendConfirmation.tsx` or should I keep the update to `useTransactionCallbacks.ts` and also update `TransactionDataInput`? Granted `TransactionDataInput` is used a lot of places.

Reply:
Ended up making getFeeEstimateDollars handle a possible null address in https://github.com/valora-inc/wallet/pull/4242/commits/8d29f198c011df360d22438e7346ea2a0fe72936!